web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -186,6 +163,21 @@ export function ActionPicker(props: { input: ReactElement }) {
   );
 }
 
+export const InputWrapper = styled(Flex).attrs({ px: 2 })`
+  row-gap: ${props => props.theme.space[2]}px;
+  column-gap: ${props => props.theme.space[2]}px;
+  align-items: center;
+  flex-wrap: wrap;
+  padding-block: 7px; // 8px - 1px = account for border
+  min-height: 38px; // input height without border
```

Comment:
> although it would be good to have some margin from the top

Ah yeah, thanks for taking care of that. I forgot to mention it yesterday, I didn't have time to figure this one out. We should strive to avoid hardcoding pixel values here but I guess it's not always possible.

For `padding-block`, could we at least use `calc` here, pick 8px from the theme and then subtract 1px from it? This way at least we're still adhering to theme-provided units.

Reply:
Done